### ISSUE TITLE
Plugin not working after last update with .NET8

### ISSUE THREAD

**Issue by Devense** [oddbear/Loupedeck.VolumeControl.Plugin#9]
As mentioned, the plugin does not work anymore after the update I've made today. Copy-paste to the Loupedeck/Plugin folder do not help, installing occures a problem with installing the plugin + crashes the Loupedeck app at the end.

**Comment by oddbear** [comment]
Hi. There has been some breaking changes for about all of my plugins in this release of Loupedeck, and I have submitted an update to the Markedplace for this one. However, it's not there yet.

Could you check out the file I attached in issue #8 ?
( https://github.com/user-attachments/files/17244417/VolumeControlPlugin.zip )

That's the one I submitted to Loupedeck, so it would be nice if you want to test it. This might break automated updates however if you install it manually, so it's a good idea to check the markedplace and install it through there when it's approved.

**Comment by Jdav00** [comment]
Ah just came here from the Wavelink plugin. I have installed this one and will be testing. Seems to be good to go for now! Thank you!!

**Comment by pr0jektcha0s** [comment]
Thank you for updating this. This is my favorite plugin.

**Comment by pr0jektcha0s** [comment]
oh, I tried it in 6 and it's not letting me using the inputs or outputs. I get action type not supported. This is under Multi-Action.

**Comment by oddbear** [comment]
There seems to be a bug in the release version of the Loupedeck software that was not there in the beta version. I am hoping they fix it in 6.1, but you will need to recreate the buttons, most likely.

**Comment by pr0jektcha0s** [comment]
I tried new buttons with no luck. Hope they fix it.

**Comment by Sochi1408** [comment]
Add a default device selection for playback and it will be the best plugin ever)

**Comment by pr0jektcha0s** [comment]
Seems the plugin still isn't working after the latest 6.0.3.21384.

**Comment by ExcitedName** [comment]
Any word on the status of this? I am using UI:6.0.5.21625;LPS:6.0.5.21571 and it is still not showing up in the marketplace.

**Comment by oddbear** [comment]
> Any word on the status of this? I am using UI:6.0.5.21625;LPS:6.0.5.21571 and it is still not showing up in the marketplace.

There is still issues with the 6.0.5 version, I have sent a bug report, but waiting for new version to test against.
Why it's not in the marketplace I have no idea. I have fallowed the submission rutine, but I have not got any feedback from Loupedeck.

**Comment by ExcitedName** [comment]
Okay, thank you for the quick response.

**Comment by balzerinyourface** [comment]
Sad day today.  It's been working like a charm until I updated the software today.  Not sure if you're still working on this project but it's my favorite thing about the loupedeck.  Just thought I'd pass it forward.

Sotware version: 6.1.1.22237

Thanks for your work on this project.

**Comment by mariano180** [comment]
I'm in the same boat, I used Loupedeck because of this plugin which was amazing.

**Comment by oddbear** [comment]
Could you try this release:
https://github.com/oddbear/Loupedeck.VolumeControl.Plugin/releases/tag/v1.0.5

Please uninstall and reinstall, as it might have breaking changes.

**Comment by pr0jektcha0s** [comment]
I've tried installing a couple times. Unfortunately, it's doing the same thing.

**Comment by oddbear** [comment]
> I've tried installing a couple times. Unfortunately, it's doing the same thing.

So the plugin is loading, but the actions won't work?
Is this only under multi action, or in general?

**Comment by pr0jektcha0s** [comment]
Sorry, should have provided more info. So I see the same issues as before where I don't see all the menu options for "set default playback" and "set default recording".  I get two condensed menus with reset options and a few audio settings, I believe they were the adjust volume inputs and outputs . If I drag them down to a multi action or a button, I get the red X.

**Comment by oddbear** [comment]
> Sorry, should have provided more info. So I see the same issues as before where I don't see all the menu options for "set default playback" and "set default recording". I get two condensed menus with reset options and a few audio settings, I believe they were the adjust volume inputs and outputs . If I drag them down to a multi action or a button, I get the red X.

If you get the plugin to load, and are able to run the actions, I am very interested to understand what is wrong here.
Would it be possible to do an online meeting to try to figure this out?
I get back from traveling this Friday, so anytime after that.

**Comment by pr0jektcha0s** [comment]
Yeah, let's do a zoom call or something.  I just returned from vacation.

**Comment by oddbear** [comment]
> Yeah, let's do a zoom call or something. I just returned from vacation.

That would be really nice. A lot easier to figure things out with more context. I am in Oslo, so GMT/UTC +2 timezone. Usually free during weekends, or after work from 16:00 here.

**Comment by mariano180** [comment]
I had to install an older version of Loudspeaker, but after a few weeks it updates and crashes again. I can't find the plugin in Market.
